<?xml version="1.0"?>
<launch>

  <include file="$(find self_drive_tasks_gazebo)/launch/q5_gazebo.launch" />

  <include file="$(find sequoia_twist_controller)/launch/sequoia_twist_controller.launch" />

  <node pkg="lidar_stop_trigger" type="lidar_stop_trigger" name="lidar_stop_trigger" >
    <remap from="scan" to="/laser/scan" />
  </node>

  <node pkg="task_nodes" type="stop_line_test" name="stop_line_test" output="screen" >
    <remap from="cmd_vel" to="/vehicle/cmd_vel" />
    <remap from="stop_line_dist" to="/camera_sim/stop_line_dist" />
  </node>

  <node ns="camera_sim" pkg="sequoia_lane_detection" type="lane_detection_node" name="lane_detection_node" output="screen" />

</launch>
